test/extended/util/framework.go

```diff
@@ -567,7 +567,7 @@ func WaitForAnImageStream(client client.ImageStreamInterface,
 
 // WaitForAnImageStreamTag waits until an image stream with given name has non-empty history for given tag.
 func WaitForAnImageStreamTag(oc *CLI, namespace, name, tag string) error {
-	waitTimeout := time.Second * 60
+	waitTimeout := time.Minute * 10
```

Comment:
yeah i think it needs to be refactored, i don't want us to wait 10 mins per imagestream in a case where dockerhub isn't available or something and things go totally off the rails.